specification/resources/resource-manager/Microsoft.Authorization/stable/2018-05-01/policyAssignments.json

```diff
@@ -618,6 +638,37 @@
       },
       "description": "List of policy assignments."
     },
+    "ManagedIdentity": {
+      "description": "The managed identity assigned to a resource.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "The type of managed identity used for the resource. 'SystemAssigned' includes both an implicitly created identity. 'None' will remove any identities from the resource.",
+          "enum": [
+            "SystemAssigned",
+            "None"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ManagedIdentityType",
+            "modelAsString": true
+          }
+        },
+        "tenantId": {
+          "description": "The tenant ID associated with the managed identity.",
```

Comment:
If you are updating the swagger, it wouldn't hurt to add a mention in the description that tenantId and principalId are guids (and which format). Probably not worth doing an update just for that.